test/e2e/singlecluster/pod_test.go

```diff
@@ -522,6 +520,20 @@ var _ = ginkgo.Describe("Pod groups", func() {
 	})
 })
 
+func expectWorkloadFinalized(ctx context.Context, k8sClient client.Client, wlKey client.ObjectKey) {
+	gomega.EventuallyWithOffset(1, func(g gomega.Gomega) error {
```

Comment:
```suggestion
	gomega.EventuallyWithOffset(1, func() error {
```

I guess that this gomega variable is not needed.

Reply:
Fixed